tests/unit_tests/test_elasticsearchstore.py

```diff
@@ -0,0 +1,156 @@
+import pytest
+import mock
```

Comment:
Please import everything you need related to mock on a single line (either remove this line, or the from mock import MagicMock)